content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -34,6 +34,12 @@ content_template: templates/task
 
 * Keeping stable etcd clusters is critical to the stability of Kubernetes clusters. Therefore, run etcd clusters on dedicated machines or isolated environments for [guaranteed resource requirements](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/hardware.md#hardware-recommendations).
 
+* The minimum recommended versions of etcd to run in production are:
+  * `3.1.11+`
+  * `3.2.10+`
+  * `3.3.0+`
```

Comment:
Should we also mention k8s releases in which those versions are recommended?

E.g. 3.2.10+ from k8s releases 1.10+ (IIRC)

@jpbetz - thoughts?

Reply:
Yes, maybe something like:

```
The minimum recommended versions of etcd to run in production are:
  * etcd `3.1.11+` for Kubernetes 1.8-1.9
  * etcd `3.2.10+` for Kubernetes 1.10+
```

?